phoebe/backend/universe.py

```diff
@@ -1762,17 +1798,21 @@ def _populate_lc(self, dataset, ignore_effects=False, **kwargs):
         """
         logger.debug("{}._populate_lc(dataset={}, ignore_effects={})".format(self.component, dataset, ignore_effects))
 
+#        print("dataset = ", dataset)  # dbg
+#        print("kwargs = ", kwargs)  # dbg
+
         lc_method = kwargs.get('lc_method', 'numerical')  # TODO: make sure this is actually passed
 
         passband = kwargs.get('passband', self.passband.get(dataset, None))
         intens_weighting = kwargs.get('intens_weighting', self.intens_weighting.get(dataset, None))
         atm = kwargs.get('atm', self.atm)
         extinct = kwargs.get('extinct', self.extinct)
         Rv = kwargs.get('Rv', self.Rv)
-        ld_mode = kwargs.get('ld_mode', self.ld_mode.get(dataset, None))
+        ld_mode = kwargs.get('ld_mode', self.ld_mode.get(dataset, 'manual'))
```

Comment:
why this change?  Is this a bugfix or does it introduce a change in behavior?

Reply:
Corrected. It was in a version without lc_params; obsolete.